workflow-service/src/main/java/com/redhat/parodos/workflow/execution/controller/WorkFlowController.java

```diff
@@ -113,6 +115,26 @@ public void updateWorkFlowCheckerTaskStatus(@PathVariable UUID workFlowExecution
 				workFlowCheckerTaskRequestDTO.getStatus());
 	}
 
+	@Operation(summary = "Restart a workflow execution with same parameters")
+	@ApiResponses(value = {
+			@ApiResponse(responseCode = "202", description = "Accepted",
+					content = { @Content(mediaType = "application/json",
+							schema = @Schema(implementation = WorkFlowExecutionResponseDTO.class)) }),
+			@ApiResponse(responseCode = "401", description = "Unauthorized", content = @Content),
+			@ApiResponse(responseCode = "403", description = "Forbidden", content = @Content),
+			@ApiResponse(responseCode = "404", description = "Not Found", content = @Content) })
+	@PostMapping("/{workFlowExecutionId}")
```

Comment:
Wouldn't be better to go for @PostMapping("/{workFlowExecutionId}**/restart**") as we might have to add other endpoints on that base url?

Reply:
I was too focused on the operationId that I missed it.
+1 for the suggestion.

Reply:
think of the need to suggest actions such as /stop or /pause.
it will be needed.

Reply:
yeah yeah but for clarity and to avoid any issue and regrets later, I'll add `/restart` to the path :)